src/bioregistry/data/curated_mappings.tsv

```diff
@@ -0,0 +1,101 @@
+subject_id	predicate_modifier	predicate_id	object_id	creator_id	comment
```

Comment:
for this to be valid SSSOM, it needs a [`mapping_justification`](https://mapping-commons.github.io/sssom/mapping_justification/) column. For all of these entries so far, the right CURIE is `semapv:ManualMappingCuration`. Later, we can also use other terms in `semapv`

we also will want to have a new set of unit tests to test the integrity of the format and content in this file.